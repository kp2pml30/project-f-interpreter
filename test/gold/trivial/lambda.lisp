(setq f (lambda ()))
(print (f))

(setq f (lambda (x y) (plus x y)))
(f 10 12)

((lambda ()
    (prog ()
     (return 10)
     11)))
