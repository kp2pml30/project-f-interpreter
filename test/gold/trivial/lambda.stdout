null
"int:22"
"int:10"
